src/gameProtocol.js

```diff
@@ -0,0 +1,22 @@
+import { gameStatus } from './constants.js'
+
+const X = 1
+const O = 0
+const EMPTY = 9
+
+export const gameStateEventArgs =
+  {
+    id: '',
+    players: [{id:'', username: '', piece:X}, {id:'', username: '', piece:O}],
+    playerTurn: '',
+    state: gameStatus.STARTED,
+    board: [EMPTY,EMPTY,EMPTY,
+      EMPTY,EMPTY,EMPTY,
+      EMPTY,EMPTY,EMPTY]
+  }
+
+export const movePlayedEventArgs =
+  {
+    player: '',
+    move: { piece: '' , position: '' }
```

Comment:
I see why we might want to have the arguments laid out as such. But since they are placeholders and used for illustrative purposes only, they should be named as such. For example: `mockGameStateEventArgs`

Reply:
And at some point we should move these placeholders to the `README.md`